utils/python/CIME/case_run.py

```diff
@@ -245,35 +245,31 @@ def DoDataAssimilation(case, da_script, lid):
 def case_run(case):
 ###############################################################################
 
-    try:
-        # Set up the run, run the model, do the postrun steps
-        run_with_submit = case.get_value("RUN_WITH_SUBMIT")
-        expect (run_with_submit,
-                "You are not calling the run script via the submit script. "
-                "As a result, short-term archiving will not be called automatically."
-                "Please submit your run using the submit script like so:"
-                " ./case.submit")
-
-        data_assimilation = case.get_value("DATA_ASSIMILATION")
-        data_assimilation_cycles = case.get_value("DATA_ASSIMILATION_CYCLES")
-        data_assimilation_script = case.get_value("DATA_ASSIMILATION_SCRIPT")
-
-        # set up the LID
-        lid = time.strftime("%y%m%d-%H%M%S")
-        os.environ["LID"] = lid
-
-        for _ in range(data_assimilation_cycles):
-            preRunCheck(case)
-            runModel(case)
-            postRunCheck(case, lid)
-            saveLogs(case, lid)       # Copy log files back to caseroot
-            getTimings(case, lid)     # Run the getTiming script
-            if data_assimilation:
-                DoDataAssimilation(case, data_assimilation_script, lid)
-
-        resubmitCheck(case)
-    except:
-        logger.warning("Exception in case_run: %s" % sys.exc_info()[1])
-        return False
-    else:
-        return True
+    # Set up the run, run the model, do the postrun steps
```

Comment:
I wanted to let except failures escape, this ensures non-zero error code without having to manage a bool all the way up the call chain.